<launch>

  <param name="initial_pose_x" type="double" value="0.0" />

  <param name="initial_pose_y" type="double" value="0.0" />
  
  <param name="initial_pose_theta" type="double" value="0.0" />


  
  <node pkg="Robotics_project1" type="vel" name="vel" output="screen"   />

  <node pkg="Robotics_project1" type="inversevel" name="inversevel" output="screen"   />
  
  <node pkg="Robotics_project1" type="odom" name="odom"  output="screen"    />

  <node pkg="tf2_ros" type="static_transform_publisher" name="world_odom" args="0 0 0 0 0 0 1 world odom "   />

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find Robotics_project1)/rviz/Robotics_project1.rviz" >
  
</launch>